backend/__fixtures__/secrets.js

```diff
@@ -176,7 +175,7 @@ const secrets = {
         'gardener.cloud/role': 'ca-cluster'
       },
       data: {
-        'ca.crt': caCrt
+        'ca.crt': 'ca.crt'
```

Comment:
I thought secret data are always base64 encoded?

Reply:
@holgerkoser the `getSecret` helper function base64 encodes the data https://github.com/gardener/dashboard/blob/7fe8183cdf282415bac9556696c9933766a2bdf2/backend/__fixtures__/secrets.js#L20-L37